### PR TITLE
Feature/alerts

### PR DESCRIPTION
**Pull request by Zoeshwa**

### CODE REVIEW

File: apps/ui/src/Communities/CommunityCard/StatusButton.tsx [hunk 1]
```diff
@@ -1,18 +1,16 @@
 import { Box, IconButton, Tooltip } from '@mui/material';
-import {
-  AddRounded,
-} from '@mui/icons-material';
+import { AddRounded } from '@mui/icons-material';
 import { UserStatus } from '@communecar/types';
 import { useCallback } from 'react';
 import { statusIcons } from '../../utils/communities/userStatusIcons';
 
 export interface CommunityCardProps {
```
Comment (Kovler10) [inline apps/ui/src/Communities/CommunityCard/StatusButton.tsx:7]: Status Button? Community Card Props?

File: apps/ui/src/Communities/CommunityCard/StatusButton.tsx [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-  setJoined: React.Dispatch<React.SetStateAction<boolean>>;
+  onJoinRequest: () => void;
```
Comment (Kovler10) [inline apps/ui/src/Communities/CommunityCard/StatusButton.tsx:8]: If this is a generic status button calling this function `onJoinRequest` when it handles leaving the community too is a bit weird

File: apps/ui/src/contexts/SnackbarContext.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      setOpen(true);
+    },
+    [],
+  );
+
+  const handleClose = (
+    _event?: Event | SyntheticEvent<any, Event>,
+    reason?: SnackbarCloseReason,
+  ) => {
+    if (reason === 'clickaway') {
```
Comment (Kovler10) [inline apps/ui/src/contexts/SnackbarContext.tsx:49]: why do we not close the snackbar on click away?

**Review by Kovler10 — CHANGES_REQUESTED** [review]
Nice! left some comments

File: apps/ui/src/Communities/CommunityCard/StatusButton.tsx (hunk 2 above)
Comment (Zoeshwa) [reply apps/ui/src/Communities/CommunityCard/StatusButton.tsx]: Let's say you changed jobs or left a place where you worked

**Pull request merged**